docs/support_metrics.md

```diff
@@ -0,0 +1,30 @@
+# Support Metrics
```

Comment:
I would call it 'Harbor compatibility list'

Reply:
Harbor Compatibility List